parse.py

```diff
@@ -44,8 +43,8 @@
 
     instr_dict = create_inst_dict(extensions, include_pseudo)
 
-    with open("instr_dict.yaml", "w") as outfile:
-        yaml.dump(add_segmented_vls_insn(instr_dict), outfile, default_flow_style=False)
+    with open("instr_dict.json", "w") as outfile:
+        json.dump(add_segmented_vls_insn(instr_dict), outfile, indent=2)
```

Comment:
@Timmmm and @aswaterman , just thinking, whether it would be a better idea to wrap the file operations in a `try-except` block for handling file writing errors _(e.g., permission issues, disk space problems)_, like below, for an example:

```
try:
    with open("instr_dict.json", "w") as outfile:
        json.dump(add_segmented_vls_insn(instr_dict), outfile, indent=2)
except IOError as e:
    print(f"Error writing to file: {e}")
```

Reply:
Yeah I agree; reading Python backtraces isn't fun. I think we should do it in another PR though probably.

Reply:
SGTM too.